strings/language/german.txt

```diff
@@ -0,0 +1,211 @@
+ah@=ach
+all@=alles
+amazing@=ausgezeichnet
+and@=und
+angel@=engel
+answer@=antwort
+apple@=apfel
+apricot@=aprikose
+attention@=achtung
+backpack@=rucksack
+bastard@=schweinehund
+bed@=bett
+beer@=bier
+best@=beste
+beutiful@=schoen
+bible@=bibel
+blood@=blut
+bomb@=bombe
+book@=buch
+books@=buecher
+bread@=brot
+broke@=brechte
+broken@=kaputt
+bullshit@=quatsch
+burger@=berger
+canon@=kanone
+capitalism@=kapitalismus
+captain@=kapitaen
+car@=auto
+cargo@=freight
+carrot@=karrote
+cars@=autos
+cat@=katze
+cats@=katzen
+cheers@=prost
+cheese@=kaese
+child@=kind
+children@=kinder
+christmas@=weinachts
+cigar@=zigarre
+clear@=klar
+coconut@=kokosnuss
+coffee@=kaffee
+cold@=kalt
+cow@=kuh
+day@=tag
+days@=tage
+delicious@=lecker
+device@=ding
+do@=tu
+doctor@=doktor
+doctors@=doktors
+does@=tut
+dog@=hound
+dogs@=hounds
+doing@=tun
+doodad@=ding
+door@=tuer
+drink@=trink
+egg@=ei
+eggs@=eier
+electricity@=elektrizitaet
+emperor@=kaiser
+end@=ende
+exactly@=genau
+fat@=fett
+father@=vater
+fire@=feuer
+fish@=fisch
+five@=fuenf
+flash@=blitz
+four@=vier
+free@=frei
+freeway@=autobahn
+fries@=pommes
+full@=voll
+gadget@=ding
+girl@=maedchen
+god@=gott
+gods@=gotter
+good@=gut
+goodbye@=auf wiedersehen
+greetings@=servus
+ground@=grund
+group@=gruppe
+half@=halb
+hat@=hut
+hazelnut@=haselnuss
+hello@=hallo
+here@=hier
+highway@=autobahn
+hotdog@=bratwurst
+house@=haus
+hundred@=hundert
+hurray@=hurra
+idiot@=dumbkopf
+idiots@=dumbkops
+inner@=innere
+is@=ist
+it@=es
+juice@=saft
+king@=koenig
+lord@=her
+maw@=maul
+meat@=fleish
+medic@=doktor
+medics@=doktors
+milk@=milch
+miss@=frau
+missus@=frau
+mister@=herr
+mother@=mutter
+mouse@=maus
+mr@=herr
+mrs@=frau
+ms@=frau
+munich@=muenchen
+nah@=nee
+net@=netz
+new@=neu
+nice@=nett
+no@=nein
+not@=nicht
+number@=nummer
+nut@=nuss
+o'clock@=uhr
+object@=objekt
+oh@=aeh
+old@=alt
+one@=eins
+orange@=orangen
+oven@=backoven
+over@=uber
+paper@=papier
+pasta@=nudeln
+people@=leute
+pepper@=pfeffer
+pistol@=pistole
+please@=bitte
+police@=polizei
+precisely@=genau
+preheat@=vorheiss
+pretty@=schoen
+pretzel@=brezel
+rat@=ratte
+red@=rot
+salad@=salat
+salt@=salz
+sauce@=sosse
+sausage@=bratwurst
+security@=polizei
```

Comment:
I know the scots accent does something similar, but we try to distance SS13 corporate security from IRL police as much as possible (remind me to change the scots one too)
```suggestion
```

Reply:
good point, I'll just remove it entirely. I opted for polizei, as an English speaker would understand it better than the german word for security.